airflow/bin/cli.py

```diff
@@ -2395,7 +2395,8 @@ def get_parser(cls, dag_parser=False):
         subparsers.required = True
 
         subparser_list = cls.dag_subparsers if dag_parser else cls.subparsers_dict.keys()
-        for sub in subparser_list:
+        sorted_subparser_list = sorted(subparser_list)
+        for sub in sorted_subparser_list:
```

Comment:
Thanks @cchung100m , a nice change to me.

But not sure if we need an intermediate variable `sorted_subparser_list` here. Can you consider directly changing `for sub in subparser_list:` into `for sub in sorted(subparser_list):`? Personally I think it's good enough and a bit more concise.